docs/concepts/vega-chain.md

```diff
@@ -143,37 +143,113 @@ Read more: [Risks of over-staked validators](/docs/concepts/vega-chain#too-much-
 
-If, at the end of an epoch, a validator does not have sufficient stake self-nominated or has overall too much stake, then their validator score will be lowered, which can impact the rewards a validator and its nominators receive. 
+This can also affect the rewards they and their nominators receive.
+If a consensus validating node stops validating, or performs poorly, then a standby validator can replace it.
+
+### How consensus validators are chosen
+Consensus validators are chosen based on a range of variables, including the validator scores, and the number of available slots. If there are no empty slots, at most one validator can be changed per epoch.
+
+If a standby validator has a better overall performance than an existing consensus validator, then a standby validator can be promoted to replace a consensus validator.
+
+## Standby validators
+Standby (also called ersatz) validators do not contribute to the chain, but are set up to join the consensus validator set if an existing validator drops off. As they don’t participate in consensus, they don’t need to be registered with a multisig contract.
+
+As with the consensus validators, standby validators are defined based on how much self-stake and nominated stake they have.
+
+Standby validators, and the tokenholders who stake them, receive a share of rewards. The rewards for standby validators are calculated and penalised in the same way as consensus validators, except scaled down. How much they are scaled is based on the network parameter `network.validators.ersatz.rewardFactor`.
+
+:::info 
+The network is set to not allow any standby validators for alpha mainnet, and the number of validators will be increased via governance as early alpha mainnet progresses.
+:::
+
+### How standby validators are chosen
+If there are free slots for one or more standby validators, and there are nodes that have submitted the transaction to join (and satisfy all joining conditions), they are added as standby validators in the next epoch. 
+
+If a node that submits the transaction to join has a higher score than the lowest scoring standby validator (scaled up by the incumbent factor), then it will become a standby validator and the lowest scoring standby validator is removed from the standby set.
+
+As the nodes vying for a standby spot will not have a performance record, their performance score is calculated as the average of the performance scores of all standby validators.
+
+Note: An inactive node that's proposing to become a validator will have a performance score of 0 and will thus be automatically excluded, regardless of their stake.
+
+Read more: [Becoming a validator](/docs/concepts/vega-chain#becoming-a-validator)
+
+### Moving from standby to consensus validator
+A standby validator that wants to be in line for promotion to become a consensus validator needs to do the following: 
+
+1. Run a non-validating node
```

Comment:
(I know it's how it's written in the spec but) this should be run a validating node.